### PR TITLE
arecord: add page

### PR DESCRIPTION
**Pull request by workflow**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/arecord.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# arecord
+
+> Sound recorder for ALSA soundcard driver.
+
+- Record a snippet in "CD" quality:
```
Comment (navarroaxel) [inline pages/linux/arecord.md:5]: I tried this, the only way to stop the recording is `Ctrl+C`?

File: pages/linux/arecord.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+`arecord -vv -f cd output.wav`
+
+- Pass a raw recording for further processing:
```
Comment (navarroaxel) [inline pages/linux/arecord.md:9]: I tried this command and this is like the 1st one but saves the audio in `mp3` format with the `| lame -r - output.mp3` the audio is output to stdout (terminal). I'm not sure about this example description, can another reviewer check?

File: pages/linux/arecord.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# arecord
+
+> Sound recorder for ALSA soundcard driver.
```
Comment (navarroaxel) [inline pages/linux/arecord.md:3]: I couldn't find a specific page for this command, but the ALSA home page is a wiki, and the only page with references to this command that I could find was this: https://www.alsa-project.org/wiki/Asoundrc.
@sbrl, do you think we should add a link to this specific page or the wiki home?

File: pages/linux/arecord.md (hunk 1 above)
Comment (workflow) [reply pages/linux/arecord.md:5]: Thanks for your swift review @navarroaxel! 

Yes, that's correct. An alternative is to specify a fixed duration using `-d` or `--duration`. I mostly use it in the dynamic style shown by this snippet, but happy to replace that with the fixed duration one if you think that's generally more useful for people.

Comment (workflow) [reply pages/linux/arecord.md]: Added explanation for `Ctrl+C` in b53cdb6406431a583c639dac1b17366d18c460f0

Comment (workflow) [reply pages/linux/arecord.md]: Added fixed duration snippet (no `Ctrl+C` required) in de200e42fdb94daaec60902d2c3a28299f38e743

File: pages/linux/arecord.md (hunk 2 above)
Comment (workflow) [reply pages/linux/arecord.md:13]: The effect should be saving the variable duration audio snippet as `output.mp3`. Doesn't happen for you?

Comment (workflow) [reply pages/linux/arecord.md]: Improved the description in d1e19443434197a014503c05b061153485d7b1de

File: pages/linux/arecord.md (hunk 3 above)
Comment (workflow) [reply pages/linux/arecord.md:3]: Could also link to the manpage at https://linux.die.net/man/1/arecord ?

File: pages/linux/arecord.md (hunk 2 above)
Comment (navarroaxel) [reply pages/linux/arecord.md]: Sorry, I had a typo I meant `without`.

File: pages/linux/arecord.md (hunk 3 above)
Comment (workflow) [reply pages/linux/arecord.md:3]: Added that in 90efa6b91c29687a0b12cf21907b644139d1e958

Comment (sbrl) [reply pages/linux/arecord.md:3]: Yeah, the man page sounds like a better idea in this case.

File: pages/linux/arecord.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://linux.die.net/man/1/arecord>.
+
+- Record a snippet in "CD" quality (finish with Ctrl-C when done):
+
+`arecord -vv -f cd {{path/to/file.wav}}`
+
+- Record a snippet in "CD" quality, with a fixed duration of 10 seconds:
+
+`arecord -vv -f -d {{duration_in_seconds}} cd {{path/to/file.wav}}`
```
Comment (sbrl) [inline pages/linux/arecord.md:12]: This doesn't quite match the description. Perhaps a specific example would aid clarity here?
```suggestion
`arecord -vv -f -d {{10}} cd {{path/to/file.wav}}`
```

Also, are there any long forms of these arguments (e.g. `-f`, `-d`, etc)? We generally refer long-form arguments in tldr pages to aid clarity.

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page, @workflow! I've left a comment below for you to review.

File: pages/linux/arecord.md (hunk 4 above)
Comment (workflow) [reply pages/linux/arecord.md]: Thanks for the review! Happy to change to long form, maybe everything except `-v`?

Comment (workflow) [reply pages/linux/arecord.md]: b2a05e043ebccf79ff4a20bd10c3cd9ca44b236a

**Review by sbrl — APPROVED** [review]
Thanks, @workflow :smiley_cat:

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**